pure-docker/deploy-codeinsights-db.sh

```diff
@@ -12,10 +12,6 @@ set -e
 VOLUME="$HOME/sourcegraph-docker/codeinsights-db-disk"
 ./ensure-volume.sh $VOLUME 999
 
-# Remove timescaledb from the shared_preload_libraries configuration
```

Comment:
This is no longer needed, it was a one-time command added during the Timescale -> Postgres migration.